### ISSUE TITLE
What the hell is this Lunar crack detected?

### ISSUE THREAD

**Issue by adithyaappu** [alin23/Lunar#474]
# Issue details
- Mac device where Lunar is installed (Macbook Pro 2019, iMac M1, Mac Mini M1, Hackintosh etc.):
    Macbook Pro 2014

- Monitor model (LG UltraFine 5K, DELL P2715Q etc.):
    *DELL P2419H*

- Monitor connection to the Mac device (HDMI-to-USB-C, USB-C-to-USB-C, miniDisplayPort-to-DisplayPort etc.):
    HDMI

- Using an USB Docking Station or Hub:
    for other purposes

- Lunar mode used (check it in the top-right corner of the Lunar interface)
    manual

- Tried Lunar Diagnostics?
> 1. Click on **Open Lunar Diagnostics** in the Lunar menu
> 2. Follow through the steps when asked

    *not relevant*

# Issue description:
*annoying flashing screen 'Lunar Crack deteccted', eventhough I don't have any. this is ridiculous.*

![image](https://user-images.githubusercontent.com/37062841/178188377-992075b2-d69f-4e72-a57c-dd31ffd9cf08.png)

**Comment by alin23** [comment]
It's a crack detection algorithm. If you modified the Lunar.app bundle in any way, this will show up. Download Lunar from the [official website](https://lunar.fyi/) and make sure to not make any modifications to the app.

**Comment by adithyaappu** [comment]
I haven't used any modification. what is triggering this? Feels like a bug to me.

**Comment by alin23** [comment]
It can also be cause by malware that infects files, or by software that alters the bundle signature. Can you please check the hash of the app in a terminal?

```sh
cat /Applications/Lunar.app/**/* 2>/dev/null | sha256sum
```

The hash for the latest version `v5.7.2` should be 

```sh
bd09fd2b138ff61cecc663879b3efd35c5490696773e154c0b14e4ca020b3202  -
```

**Comment by adithyaappu** [comment]
will do. anyway I'm re-installling the app after cleaning junks. Will test after it.

**Comment by adithyaappu** [comment]
I have a doubt. will this happen if I haven't updated the app to the latest version? I kinda remember this happening before and then stopped showing when I updated the app. I had the previous version.

**Comment by alin23** [comment]
Yes, there were some bugs related to this detection giving false positives on previous versions, especially if the light sensor firmware installer had been used. Everything should be fixed in 5.7.2
